modin/backends/pandas/parsers.py

```diff
@@ -68,6 +156,22 @@ def get_dtypes(cls, dtypes_ids):
 
     @classmethod
     def single_worker_read(cls, fname, **kwargs):
+        """Perform reading by single worker (default-to-pandas implementation).
+
+        Parameters
+        ----------
+        fname: str, path object or file-like object
+            Name of the file or file-like object to read.
+
+        Returns
+        -------
+        BaseQueryCompiler,
+        dict/OrderedDict or
+        pandas.io.parsers.TextFileReader:
```

Comment:
are we returning one of these three, or all three at the same time? docstring looks ambiguous here, pls clear it up somehow

Reply:
Fixed.